### PR TITLE
Justinxue/bre 21 implement dynamic routing for event descriptions

### PR DESCRIPTION
**Pull request by jxmoose**
[//]: # "Feel free to customize this template to your project's vibes"

## :bread: What's new in this PR

Adding event page for individual events that show various info

### :croissant: Description
[//]: # "Required - Describe what's new in this PR in a few lines. A description and bullet points for specifics will suffice."

New queries to fetch facility by id, event by id, dynamic routing event page which is linked to by the /activeEvents page

Some issues:
1. Hard coded "prefers orchestra" under facility name
2. the signup has no functionality
3. loading is slow but i think thats supabase

### :baguette_bread: Screenshots
[//]: # "Required for frontend changes, otherwise optional but strongly recommended. Add screenshots of expected behavior - GIFs if you're feeling fancy! Use the provided image template. Drag the desired image into the PR, then copy the link into the placeholder."

image

image

[image placeholder]: 

## :sandwich: How to review
[//]: # 'Required - Describe the order in which to review files and what to expect when testing locally. Is there anything specifically you want feedback on? Should this be reviewed commit by commit, or all at once? What are some user flows to test? What are some edge cases to look out for?'

Just look at the page /activeEvents and click on any of the events

## :pie: Next steps
[//]: # "Optional - What's NOT in this PR, doesn't work yet, and/or still needs to be done. Note any temporary fixes in this PR that should be cleaned up later."

### :pretzel: Online sources
[//]: # 'Copy links to any tutorials or documentation that was useful to you when working on this PR'

### :bagel: Related PRs
[//]: # "Add related PRs you're waiting on/ PRs that will conflict, etc; if this is a refactor, feel free to add PRs that previously modified this code"

CC: @celinechoiii

### CODE REVIEW

File: app/activeEvents/[event_id]/page.tsx
Comment (celinechoiii) [inline app/activeEvents/[event_id]/page.tsx:1]: rename `activeEvents` to `discover`

**Review by celinechoiii — CHANGES_REQUESTED** [review]

File: app/discover/[event_id]/styles.ts [hunk 1]
```diff
@@ -0,0 +1,134 @@
...
+
+export const ShowInterest = styled(H5)`
+  margin-top: 2.5rem;
+  font-weight: 500;
+  margin-bottom: 1.5rem;
+`;
+
+export const InterestBlock = styled.div`
+  padding: 0.75rem 1rem;
+  width: 20rem;
```
Comment (celinechoiii) [inline app/discover/[event_id]/styles.ts]: the width of the interest block is short on mobile

File: app/discover/[event_id]/page.tsx [hunk 2]
```diff
@@ -0,0 +1,132 @@
...
+            <Bullet>Host should be able to carry 15lbs of equipment</Bullet>
+          </div>
+        ) : (
+          ''
+        )}
+        <ShowInterest> Show Interest </ShowInterest>
+        {InterestBlockGen(
+          'To Perform',
+          'Be the star of the show!',
+          '/images/help.svg',
```
Comment (celinechoiii) [inline app/discover/[event_id]/page.tsx]: you got the SVGs mixed up LOL

File: app/discover/[event_id]/styles.ts [hunk 3]
```diff
@@ -0,0 +1,134 @@
...
+  margin-top: 0.875rem;
+`;
+
+export const Location = styled(SMALL)`
+  color: #6d6d6d;
+  margin-top: 0.375rem;
+`;
+
+export const IndividualTag = styled.span`
+  background-color: #6d6d6d;
```
Comment (celinechoiii) [inline app/discover/[event_id]/styles.ts]: pls use global colors!

**Review by celinechoiii — CHANGES_REQUESTED** [review]

**Comment by celinechoiii** [comment]
LGTM!!!
